Functions-request-config.js

```diff
@@ -6,19 +6,19 @@ require("@chainlink/env-enc").config()
 
 // Configure the request by setting the fields below
 const requestConfig = {
-  // Location of source code (only Inline is currently supported)
-  codeLocation: Location.Inline,
-  // Location of secrets (only Remote or DONHosted is supported)
-  secretsLocation: Location.DONHosted,
-  // Code language (only JavaScript is currently supported)
-  codeLanguage: CodeLanguage.JavaScript,
   // String containing the source code to be executed
-  //source: fs.readFileSync("./calculation-example.js").toString(),
+  // source: fs.readFileSync("./calculation-example.js").toString(),
```

Comment:
have added the requirement + instructions to set the COINBASE API key in the README as part of the quick start, as I think that is the path developers need to get to ASAP rather than simple computations. When i investigate down-sizing to a single source in the examples this may be moot. for now leaving as is.